pkg/cmd/cli/cmd/examples.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/spf13/cobra"
+)
+
+const examplesLong = `Displays one big list of examples for most commands available in the OpenShift client tools.
```

Comment:
If the issue is having it *exposed* in help, one option is to remove the description, like we do in `openshift infra`, which will remove it from help everywhere.

----- Original Message -----
> > @@ -0,0 +1,81 @@
> > +package cmd
> > +
> > +import (
> > +	"fmt"
> > +	"io"
> > +
> > +	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
> > +	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
> > +	"github.com/spf13/cobra"
> > +)
> > +
> > +const examplesLong = `Displays one big list of examples for most commands
> > available in the OpenShift client tools.
> 
> I don't personally find pages and pages of unrelated examples useful in the
> CLI
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3325/files#r33302910